packages/react-components/components/input/stories/textarea/textarea.stories.mdx

```diff
@@ -133,6 +133,28 @@ A textarea can have a minimum number of rows.
     </Story>
 </Preview>
 
+### Size
+
+A textarea can vary in size.
+
+Available sizes are: `"small"`, `"medium"` and `"large"`.
+
+<Preview>
+    <Story name="size">
+        <div className="flex flex-row items-end items-space-between">
+            <div className="flex-auto">
```

Comment:
Do you need flex-auto here?

In fact, I added it for the date picker but I don't understand why we need `flex-auto.`

Reply:
See flex auto explanation on MDN : The item is sized according to its width and height properties, but grows to absorb any extra free space in the flex container,  this is the equivalent of "flex: 1 1 auto"

soit flex grow 1 shrink1 basis auto